Tests/JournalMonitorTests.cs

```diff
@@ -474,10 +474,8 @@ public void TestJournalMissionAccepted1()
         public void TestFriends()
         {
             string line = "{ \"timestamp\":\"2017-08-24T17:22:03Z\", \"event\":\"Friends\", \"Status\":\"Online\", \"Name\":\"_Testy_McTest_\" }";
-            string line2 = "{ \"timestamp\":\"2017-08-24T17:22:03Z\", \"event\":\"Friends\", \"Status\":\"Offline\", \"Name\":\"_Testy_McTest_\" }";
 
             List<Event> events = JournalMonitor.ParseJournalEntry(line);
-            events = JournalMonitor.ParseJournalEntry(line2);
```

Comment:
Doesn't this change break the test?

Reply:
My mistake.

Reply:
Well this is intgeresting. It turns out that the comment is wrong and both parses generate an event if you test them. I'll amenmd the test but then I'd appreciate your review again please!

Reply:
Yeah I noticed that too. The intent of the test isn't being met if you can take away a fundamental part of it and still get a passing result.